core/Plugin/Visualization.php

```diff
@@ -233,6 +236,35 @@ protected function buildView()
         return $view;
     }
 
+    /**
+     * @internal
+     */
+    protected function loadDataTableFromAPI()
```

Comment:
We have to request the API method manually as we want to disable the postProcessor in ResponseBuilder.